docs/conf.py

```diff
@@ -19,8 +19,7 @@
 import sys
 from pkg_resources import get_distribution
 
-sys.path.insert(0, str(Path('../src').resolve()))
-sys.path.insert(1, str(Path('./exts').resolve()))
+sys.path.insert(0, str(Path('./exts').resolve()))
```

Comment:
@kratsg Is it possible for us to get rid of the path insertion entirely like @henryiii suggested? Put another way, can you remind me why we need this?

Reply:
We need the xrefs: https://github.com/michaeljones/sphinx-xref -- this isn't a python package, so we are manually pulling it in for now.